integration/tests/cook/test_cli.py

```diff
@@ -1944,7 +1944,7 @@ def main():
                 self.assertEqual('Hello world!', cli.decode(cp.stdout))
 
     def test_usage(self):
-        command = 'sleep 300'
+        command = f'sleep {util.DEFAULT_TEST_TIMEOUT_SECS}'
```

Comment:
This change is unrelated to the rest of the PR, but the test flaked on this branch, so I'm fixing it here.